composer/core/time.py

```diff
@@ -195,6 +197,12 @@ def _parse(self, other: object) -> Time:
         raise NotImplementedError(f"Cannot convert type {other} to {self.__class__.__name__}")
 
     def _cmp(self, other: object) -> int:
+        # When doing comparisions, and other is an integer (or float), we can safely infer
+        # the unit from self.unit
+        # E.g. calls like this should be allowed: if batch < 42: do_something()
+        # This eliminates the need to call .value everywhere
+        if isinstance(other, (int, float)):
+            other = type(self)(other, self.unit)
```

Comment:
I prefer this style should someone subclass the `Time` -- then it will return the same type as the subclass, rather than the parent class. Granted this depends on the constructor being the same...